src/components/BackButton/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import './style.css';
+
+const BackButton = () => {
+  const history = useHistory();
+
+  return (
+    <div className="ml-5">
+      <button className="back-button" onClick={history.goBack}>Back</button>
+    </div>
+  )
+}
+
+export default BackButton;
```

Comment:
Nice to have a back button implemented, would be nice if it took us back to the list of repositories for a user instead of having to search again.